lib/eksekuter.rb

```diff
@@ -38,17 +37,9 @@ def stderr
     @stderr ||= StringIO.new('', 'r')
   end
 
-  def stdouterr
-    @stdouterr ||= StringIO.new('', 'r')
-  end
-
   def spawn_process
-    @stdout = @stderr = @stdouterr = nil
-    if called_methods.include? :stdouterr
-      @stdin, @stdouterr, @wait_thr = Open3.popen2e(cmd, opts)
-    else
-      @stdin, @stdout, @stderr, @wait_thr = Open3.popen3(cmd, opts)
-    end
+    @stdout = @stderr = nil
```

Comment:
We don't need this line anymore. It was meant to initialise any of the three fields which may be _not_ initialized when calling `popen2e` vs `popen3`.